CloudFlare/cloudflare.py

```diff
@@ -438,6 +488,7 @@ def __init__(self, email=None, token=None, certtoken=None, debug=False, raw=Fals
         self._base = self._v4base(email, token, certtoken, base_url, debug, raw)
 
         # add the API calls
-        api_v4(self)
+        api_v4(self)  # todo: This should really be part of the class?
```

Comment:
@wslippey Unless we're trying to futureproof api_v5, etc... which seems strange with such a small footprint. And AFAIK CloudFlare doesn't run multiple API's, so for both those reasons this seems superfluous.